CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## v0.110.0 [2023-07-13]
+
+_What's new?_
+
+- New tables added
+  - [aws_appstream_fleet](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_appstream_fleet) ([#1838](https://github.com/turbot/steampipe-plugin-aws/pull/1838))
+
+_Enhancements_
+
+- Added the `event_topics` and `snapshot_limit` columns to the `aws_directory_service_directory` table. ([#1833](https://github.com/turbot/steampipe-plugin-aws/pull/1833))
+
+_Bug fixes_
+
+- Fixed the `aws_dlm_lifecycle_policy` table to correctly handle the `ResourceNotFoundException` error. ([#1834](https://github.com/turbot/steampipe-plugin-aws/pull/1834))
```

Comment:
```suggestion
- Fixed the `aws_dlm_lifecycle_policy` table to correctly return results instead of an error. ([#1834](https://github.com/turbot/steampipe-plugin-aws/pull/1834))
```